src/assets/svgs/frameworks/angular.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M13.3761 16.864H16.0961H18.8161L16.0961 10.32L13.3761 16.864Z" fill="#D6E4F7"/>
```

Comment:
Fill should be ` fill="currentColor"` so we can control the color more easily. This applies to all the SVGs added

Reply:
Where do we define `currentColor`?

Reply:
It's a convention thing, I see. The Vue logo has two colours in it, how do you suggest I handle that? 
![image](https://user-images.githubusercontent.com/5170590/101949429-76496c80-3ba8-11eb-84d1-5cbd0167b4a2.png)

Reply:
The fill would be on the <path> (end of line 2 in that example).  The two-tone one is a bit of a challenge. I suppose we could do a `fill="currentcolor"` and  a `fill-opacity="0.4"` and try to match it as close as we can.

Reply:
How would you feel about me making the second line currentColor and leaving the other one the way it is? There'll be a phase 2 of this PR at some point in the future to make the logos configurable

Reply:
Actually your suggestion works perfectly, I'm going to use that